integration_tests/dbt_project.yml

```diff
@@ -0,0 +1,71 @@
+name: 'amazon_selling_partner_integration_tests'
+version: '0.1.0'
+config-version: 2
+
+profile: 'integration_tests'
+
+dispatch:
+  - macro_namespace: dbt_utils
+    search_order: ['spark_utils', 'dbt_utils']
+
+vars:
+  amazon_selling_partner:
+    amazon_selling_partner_schema: amazon_selling_partner_integration_tests_1
```

Comment:
This needs to be a smaller name in order for postgres to succeed

Reply:
adjusted and it's working 🎉